iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AttachLoadOnVoltageLevel.java

```diff
@@ -0,0 +1,171 @@
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.BusbarSectionPosition;
+
+import java.util.List;
+
+import static com.powsybl.iidm.modification.topology.TopologyModificationUtils.*;
+
+/**
+ * This method adds a new load on an existing voltage level. The voltage level should be described
+ * in node/breaker.
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class AttachLoadOnVoltageLevel implements NetworkModification {
+
+    //Load attributes
+    private final String loadId;
+    private final double p0;
+    private final double q0;
+    private final LoadType loadType; //add constant
+
+    //VoltageLevel attributes
+    private final String voltageLevelId;
+    private String bbsId; //Id of the busBar where the switch will be open
+
+    private List<String> bbsIds;
+
+    public AttachLoadOnVoltageLevel(String loadId, double p0, double q0,
+                                    LoadType loadType, String voltageLevelId, String bbsId) {
+        this.loadId = loadId;
+        this.p0 = p0;
+        this.q0 = q0;
+        this.loadType = loadType;
+        this.voltageLevelId = voltageLevelId;
+        this.bbsId = bbsId;
+    }
+
+    public AttachLoadOnVoltageLevel(String loadId, double p0, double q0,
+                                    LoadType loadType, String voltageLevelId, List<String> bbsIds) {
+        this.loadId = loadId;
+        this.p0 = p0;
+        this.q0 = q0;
+        this.loadType = loadType;
+        this.voltageLevelId = voltageLevelId;
+        this.bbsIds = bbsIds;
+    }
+
+    public AttachLoadOnVoltageLevel(String loadId, double p0, double q0,
+                                    LoadType loadType, String voltageLevelId) {
+        this.loadId = loadId;
+        this.p0 = p0;
+        this.q0 = q0;
+        this.loadType = loadType;
+        this.voltageLevelId = voltageLevelId;
+    }
+
+    public String getLoadId() {
+        return loadId;
+    }
+
+    public double getP0() {
+        return p0;
+    }
+
+    public double getQ0() {
+        return q0;
+    }
+
+    public LoadType getLoadType() {
+        return loadType;
+    }
+
+    public String getVoltageLevelId() {
+        return voltageLevelId;
+    }
+
+    public String getBbsId() {
+        return bbsId;
+    }
+
+    public void setBbsId(String bbsId) {
+        this.bbsId = bbsId;
+    }
+
+    public List<String> getBbsIds() {
+        return bbsIds;
+    }
+
+    public void setBbsIds(List<String> bbsIds) {
+        this.bbsIds = bbsIds;
+    }
+
+    private void createTopologyAutomatically(Network network, VoltageLevel voltageLevel, int loadNode, int forkNode) {
+        BusbarSection bbs = network.getBusbarSection(bbsId);
+        int bbsNode = bbs.getTerminal().getNodeBreakerView().getNode();
+        createNodeBreakerSwitches(loadNode, forkNode, bbsNode, loadId, voltageLevel.getNodeBreakerView());
+        if (voltageLevel.getNodeBreakerView().getBusbarSectionCount() > 1) {
```

Comment:
Is the other case normal? Maybe we should at least write a warning (to be integrated with Reporter later)